### PR TITLE
Added grocery store search terms

### PR DESCRIPTION
**Pull request by TheAdventurer64**

### CODE REVIEW

File: data/presets/presets/shop/supermarket.json [hunk 1]
```diff
@@ -12,7 +12,11 @@
     "terms": [
         "grocery",
         "store",
-        "shop"
+        "shop",
+        "grocery store"
```
Comment (quincylvania) [inline data/presets/presets/shop/supermarket.json:16]: The build failed because you're missing commas after these values, but I'll just add them for you.

**Comment by quincylvania** [comment]
@TheAdventurer64 Thanks, welcome to open source development!

One tip, you can run `npm run build` to check for errors and generate the derived data for your changes.

File: data/presets/presets/shop/supermarket.json (hunk 1 above)
Comment (TheAdventurer64) [reply data/presets/presets/shop/supermarket.json:16]: Thanks!

**Comment by TheAdventurer64** [comment]
How exactly do you run commands?

**Comment by quincylvania** [comment]
> How exactly do you run commands?

From a command line tool like Terminal on macOS or cmd.exe on Windows. If you haven't used one before, there are lots of [tutorials](https://www.webfx.com/blog/web-design/command-line/). The command line is an important part of developing most software, including NodeJS apps like iD.

**Pull request merged**